rewrite-java-tck/src/main/java/org/openrewrite/java/tree/LombokTest.java

```diff
@@ -821,6 +826,107 @@ public void test() {
     @SuppressWarnings("MismatchedReadAndWriteOfArray")
     @Nested
     class LessSupported {
+        /*
+         java 8 cannot figure out all type checking:
+         - When the @AllArgsConstructorHandler, @NoArgsConstructorHandler and @NoArgsConstructorHandler annotations are
+           used with the `onConstructor_` param, Lombok does not call the JavacAnnotationHandlers.
+         - The @Jacksonized annotation does somehow turns into `ClassDeclaration->CompilationUni` error
+         */
+
+        @Test
+        // TODO: Find solution and remove this test
+        void jacksonizedForJava8() {
+            rewriteRun(
+              spec -> spec
+                .parser(JavaParser.fromJavaVersion().classpath("jackson-annotations", "lombok"))
+                .typeValidationOptions(TypeValidation.none()),
```

Comment:
Can we be more specific here that `none`? Something like `all()` followed by explicit disabling select checks? That might then avoid regressions.

Reply:
A little bit, I updated the code.

Reply:
Thanks! Can we fix the conflicts after the trailing comma fixes as well?